library/api.inc

```diff
@@ -35,8 +35,9 @@ function formSubmit ($tableName, $values, $id, $authorized = "0")
         // Bring in $sanitize_all_escapes variable, which will decide
         // the variable escaping method.
         global $sanitize_all_escapes;
+		global $attendant_type;
```

Comment:
How is $attendant_type set. Can it be manipulated by the user via a $REQUEST (just seeing if need to run below column throught the column sql escaping mechanism:
https://github.com/openemr/openemr/blob/master/library/formdata.inc.php#L82